packages/react-integration/demo-app-ts/src/components/demos/DonutUtilizationChartDemo/DonutUtilizationGreenStaticRightDemo.tsx

```diff
@@ -42,7 +42,7 @@ export class DonutUtilizationGreenStaticRightDemo extends React.Component<{}, {
           >
             <ChartDonutUtilization
               data={{ x: 'Storage capacity', y: used }}
-              labels={({ datum }) => (datum.x ? `${datum.x}: ${datum.y}%` : null)}
+              labels={({ datum }) => (datum.x ? `${datum.x}: ${datum.y}%` : '')}
```

Comment:
This is suppose to be null to prevent the tooltip from displaying in certain scenarios. 

It's possible that it could be undefined, but an empty string is not what we want here. An empty string will cause the tooltip to display an empty box.

Reply:
Got it - thanks for the feedback here, and in the related issues!  I'll update the type to allow for null and revert these back

Reply:
Reverted this